loadtest/loadtest.mjs

```diff
@@ -1,5 +1,26 @@
-// NOTE: This file assumes one of {staging,production}.js is available during execution.
-definition.setTarget($host);
+/*
+ * NOTICE: DO NOT EDIT!
+ *
+ * This file is managed via https://github.com/stormforger/example-github-actions/
+ */
+
+var config;
+
+// Define the target environment via --define ENV=\"staging\"
+if (ENV == "staging") {
+  config = {
+    dsPrefix: "example-github-actions/staging/",
+    host: "http://testapp.loadtest.party:9001",
+  }
+} else if (ENV == "production") {
+  config = {
+    dsPrefix: "example-github-actions/production/",
+    host: "https://testapp.loadtest.party",
+  }
```

Comment:
We should have a `else` here, where we `throw new Error("Unknown environment")` or something.

Reply:
done